app/src/main/java/org/mifos/mobile/ui/recent_transactions/RecentTransactionViewModel.kt

```diff
@@ -1,4 +1,4 @@
-package org.mifos.mobile.viewModels
+package org.mifos.mobile.ui.recent_transactions
 
```

Comment:
create a bool mutable stateflow for refreshing the screen in viewmodel , create a function for it using launch which will set to false after fetching transactions